package.json

```diff
@@ -24,13 +24,15 @@
     "clsx": "^2.1.0",
     "geist": "^1.2.2",
     "immer": "^10.1.1",
+    "install": "^0.13.0",
     "lucide-react": "^0.312.0",
     "next": "^14.1.1",
     "postcss": "8.4.29",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-hook-form": "^7.51.3",
     "react-hot-toast": "^2.4.1",
+    "react-icons": "^5.2.1",
```

Comment:
the lucide-icons were a plain white color. i installed react-icons because it had precolored icons. I can uninstall this and use lucide icons in place.

Reply:
I would go that route to ensure we're using the same icons throughout the app, unless the team comes to a decision otherwise.

You can color the lucide icon using tailwind classes like I did in the Nav component:

```tsx
<Menu className="text-zinc-600 dark:text-white" />
```

Let me/others know if you run into any issues!

Reply:
@alexappleget let's make sure we remove react-icons.